man/man8/swtpm.pod

```diff
@@ -406,6 +406,15 @@ Display usage info.
 
 =back
 
+=head1 NOTES
+
+If a TPM 2 is used, the user is typically required to send a TPM2_Shutdown()
+command to a TPM 2 to avoid possibly increasing the TPM_PT_LOCKOUT_COUNTER
+that may lead to a dictionary attack lockout upon next startup
+(TPM2_Startup()) of the TPM 2. Whether the TPM_PT_LOCKOUT_COUNTER is
+actually increased depends on previous commands sent to the TPM 2 as well as
+internal state of the TPM 2. Swtpm will make a best-effort and send a
+TPM2_Shutdown(SU_STATE) or TPM2_Shutdown(SU_CLEAR) if found necessary.
```

Comment:
Please be more specific on when this happen, otherwise a casual reader will wonder and need to look at the sources.

Reply:
I updated the text now with an example.